server/controllers/article.controller.js

```diff
@@ -40,5 +40,31 @@ class ArticleController {
     }
     return Response(res, 201, successMessage, { article });
   }
+
+  static async getAllArticles(req, res) {
+    /**
+     * @description - Controller to get all articles endpoint
+     *
+     * @static
+     * @param {object} request
+     * @param {object} response
+     * @returns {object}
```

Comment:
The return docs specification looks empty

Reply:
Do you mind making this comment clearer?

Reply:
` @returns {object}` does not specify what exactly is returning